pkg/reconciler/autoscaling/hpa/resources/hpa_test.go

```diff
@@ -64,11 +64,14 @@ func TestMakeHPA(t *testing.T) {
 		want: hpa(
 			withAnnotationValue(autoscaling.MetricAnnotationKey, autoscaling.CPU),
 			withAnnotationValue(autoscaling.TargetAnnotationKey, "50"),
-			withMetric(autoscalingv2beta1.MetricSpec{
-				Type: autoscalingv2beta1.ResourceMetricSourceType,
-				Resource: &autoscalingv2beta1.ResourceMetricSource{
-					Name:                     corev1.ResourceCPU,
-					TargetAverageUtilization: ptr.Int32(50),
+			withMetric(autoscalingv2beta2.MetricSpec{
+				Type: autoscalingv2beta2.ResourceMetricSourceType,
+				Resource: &autoscalingv2beta2.ResourceMetricSource{
+					Name: corev1.ResourceCPU,
+					Target: autoscalingv2beta2.MetricTarget{
+						Type:               "Utilization",
```

Comment:
Should use the same constant as above (throughout this file)

Reply:
:+1: